.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
@KarolS What is your opinion on environment specific configs (`.vscode`, `.scalafmt.conf`, etc)?